example/register_device/lib/ThingooConnector/ThingooConnector.cpp

```diff
@@ -0,0 +1,81 @@
+/***************************************************************
+   esp8266Thingoo is a library for the ESP8266/Arduino platform
+   to communicate with Thingoo API
+   Built for KNI Project "Thingoo" https://github.com/ThingooKNI/
+ **************************************************************/
+
+#include "ThingooConnector.h"
+
+#include <Arduino.h>
+#include <WiFiClientSecure.h>
+#include <ESP8266HTTPClient.h>
+#include <ArduinoJson.h>
+#include <WiFiClientSecureBearSSL.h>
+
+ThingooConnector::ThingooConnector(const char* host)
+{
+
```

Comment:
Make sure that passes `host` doesn't contain trailing slash. Otherwise it will break during http requests